### ISSUE TITLE
\dx command

### ISSUE THREAD

**Issue by apacala** [dbcli/pgcli#515]
Hi! Great work on the client, much more enjoyable than using raw `psql` ❤️.

However, I've just found out that `/dx` command is not supported:

```
app_dev> \dx
syntax error at or near "\"
LINE 1: \dx
        ^

Time: 0.002s
app_dev>
```

Would it be possible to make `/dx` work in the future versions of Pgcli?

Thanks!

**Comment by darikg** [comment]
I'll take a stab at this in the coming week or two. I've been meaning to write a guide illustrating how to add special commands in general so we can make it easier to crowdsource adding in all the other other \commands we're missing

**Comment by amjith** [comment]
@j-bennet I remember you wrote a guide a while back. I can't find it now.
Can you point Darik to that guide so he can update it if possible?

**Comment by j-bennet** [comment]
Yes, it's in pgspecial docs:

https://github.com/dbcli/pgspecial/blob/master/DEVELOP.rst

**Comment by darikg** [comment]
Oh wow that doc is fantastic, thanks @j-bennet

**Comment by j-bennet** [comment]
Probably needs to be updated now, would be great if you want to do it @darikg.

**Comment by amjith** [comment]
This is now available in v1.0. Please upgrade. 

```
pip install -U pgcli
```

**Comment by apacala** [comment]
Thank you! ❤️
